cli/pcluster/resources/batch/docker/scripts/mount_nfs.sh

```diff
@@ -74,7 +74,7 @@ mount_nfs() {
 # main function
 main() {
     master_ip=${1}
-    shared_dir=${2}
+    shared_dir="/${2}"
```

Comment:
can you optionally add this if the argument doesn't begin with a "/"?

Reply:
https://unix.stackexchange.com/questions/12283/unix-difference-between-path-starting-with-and/12285
I think in most cases "//" is the same as "/", but if you want I can work on this